client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -46,14 +59,28 @@ const ResultsTable = styled(HTMLTable).attrs({
   }
 
   /* Exclude edit buttons from copy/paste */
-  th:first-child,
-  td:first-child {
+  th:last-child,
+  td:last-child {
```

Comment:
This was a bug I just happened to notice. It only impacts manual user selection copy, not the copy button, so that's probably why it wasn't caught. I don't think it's easy to test in an automated way unfortunately